README.md

```diff
@@ -563,7 +563,36 @@ The universal object is where you define all of the custom metadata associated w
 
 **expirationDate**: The date when the content will not longer be available or valid. Currently, this is only used for Spotlight indexing but will be used by Branch in the future.
 
+
+### Tracking User Actions and Events
+Use `BranchEvent` class to track special user actions or application specific events beyond app installs, opens, and sharing . You can track events such as when a user adds an item to an on-line shopping cart, or searches for a keyword etc.
+`BranchEvent` provides interface to add contents represented by BranchUniversalObject in order to associate contents with events.
```

Comment:
BranchEvent provides an interface to add content(s) represented by a BranchUniversalObject in order to associated content(s) with events. 

(Added *an* interface, *a* BranchUniversalObject)